dev-docs/explanations/translations.md

```diff
@@ -0,0 +1,66 @@
+# Translations
+
+Ubuntu Pro Client is a special package in Ubuntu: it is updated frequently in all Ubuntu LTS releases.
+That makes language packs unsuitable for providing and updating translations for Ubuntu Pro Client.
+
+In addition, Ubuntu Pro Client includes lots of language with potential commercial and contractual
+implications related to Canonical's Ubuntu Pro offering. That means we need to maintain tight control
+over the language used in all messages.
```

Comment:
"We need to maintain tight control" sounds a bit "off". I'd suggest something more like:

That means Canonical must be both careful and specific about the language used in all messages to ensure that we do not provide misleading or incorrect information.